### PR TITLE
move initial part dist type flag to ini

### PR DESCRIPTION
**Pull request by awirb**

### CODE REVIEW

File: avaframe/com1DFA/com1DFA.py [hunk 1]
```diff
@@ -650,7 +645,7 @@ def initializeParticles(cfg, releaseLine, dem, logName=''):
             hCell = relRaster[indRely, indRelx]
             volCell = areaRaster[indRely, indRelx] * hCell
             massCell = volCell * rho
-            xpart, ypart, mPart, nPart = placeParticles(massCell, indRelx, indRely, csz, massPerPart, rng)
+            xpart, ypart, mPart, nPart = placeParticles(massCell, indRelx, indRely, csz, massPerPart, rng, cfg['initPartDistType'])
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:648]:
```suggestion
            xpart, ypart, mPart, nPart = placeParticles(massCell, indRelx, indRely, csz, 
                                                                                massPerPart, rng, cfg['initPartDistType'])
```

**Review by fso42 — CHANGES_REQUESTED** [review]
Just a minor change, please merge afterwards

File: avaframe/com1DFA/com1DFA.py [hunk 2]
```diff
@@ -821,11 +819,11 @@ def placeParticles(massCell, indx, indy, csz, massPerPart, rng):
     # TODO make this an independent function
     #######################
     # start ###############
-    if flagSemiRand:
+    if initPartDistType == 'semiRandom':
```
Comment (matthiasto) [inline avaframe/com1DFA/com1DFA.py:822]: maybe make it case unsensitive

**Review by matthiasto — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/com1DFA.py (hunk 2 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFA.py]: done

File: avaframe/com1DFA/com1DFA.py (hunk 1 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFA.py]: done

**Pull request merged**